honeybee_schema/energy/schedule.py

```diff
@@ -239,6 +239,25 @@ class ScheduleRulesetAbridged(NamedEnergyBaseModel):
     )
 
 
+class ScheduleRuleset(ScheduleRulesetAbridged):
+    """Used to define a schedule for a default day, further described by ScheduleRule."""
+
+    type: constr(regex='^ScheduleRuleset$') = 'ScheduleRuleset'
+
+    day_schedules: List[ScheduleDay] = Field(
```

Comment:
Please remove the `day_schedules` here since this property is exactly the same on the `ScheduleRulesetAbridged`